lib/pages/contact/contact.dart

```diff
@@ -1,17 +1,38 @@
 import 'dart:collection';
 import 'package:alphabet_scroll_view/alphabet_scroll_view.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_mobx/flutter_mobx.dart';
 import 'package:onestop_dev/globals/my_colors.dart';
 import 'package:onestop_dev/globals/my_fonts.dart';
+import 'package:onestop_dev/models/contacts/contact_details.dart';
 import 'package:onestop_dev/models/contacts/contact_model.dart';
 import 'package:onestop_dev/services/data_provider.dart';
+import 'package:onestop_dev/stores/contact_store.dart';
 import 'package:onestop_dev/widgets/ui/list_shimmer.dart';
+import 'package:provider/provider.dart';
 import 'contact_detail.dart';
 import 'package:onestop_dev/widgets/contact/contact_search_bar.dart';
 
-// SplayTreeMap<String, ContactModel> people= SplayTreeMap();
-// SplayTreeMap<String, ContactModel> people_search= SplayTreeMap();
-List<String> alphabets = [  "A",  "B",  "C",  "D",  "E",  "F", "I",  "L",  "M",  "N",  "O",  "P",  "Q",  "R",  "S",  "T",  "U",  "V"];
+List<String> alphabets = [
+  "A",
+  "B",
+  "C",
+  "D",
+  "E",
+  "F",
+  "I",
+  "L",
+  "M",
+  "N",
+  "O",
+  "P",
+  "Q",
+  "R",
+  "S",
+  "T",
+  "U",
+  "V"
+];
```

Comment:
@Hareesh-Nandigrama What about the missing alphabets in this list? Ideally, we want an alphabet header to appear only for the contacts that exist, but we can't hardcode the alphabets for the dummy data. Try finding a way to make the appropriate headers appear irrespective of data.

Reply:
Done